lib/utils/contributor-utils.ts

```diff
@@ -1,6 +1,13 @@
 export function getTopContributorLanguages(contributor: DbUser) {
   // Note that it's possible for a user to have no languages
-  const languages = Object.entries(contributor.languages).map(([language]) => language);
+  const languages: { language: string; percentageUsed: number }[] = Object.entries(contributor.languages).map(
```

Comment:
refactor (blocking): no need to explicitly type the variable.

```suggestion
  const languages = Object.entries(contributor.languages).map(
```